src/spring-cloud/azext_spring_cloud/_transformers.py

```diff
@@ -31,17 +31,20 @@ def transform_app_table_output(result):
         item['Public Url'] = item['properties']['url']
 
         if 'activeDeployment' in item['properties']:
-            item['Status'] = item['properties']['activeDeployment']['properties']['status']
-            item['Instance Count'] = item['properties']['activeDeployment']['properties']['deploymentSettings']['instanceCount']
-            item['CPU'] = item['properties']['activeDeployment']['properties']['deploymentSettings']['cpu']
-            item['Memory'] = item['properties']['activeDeployment']['properties']['deploymentSettings']['memoryInGb']
+            isStarted = item['properties']['activeDeployment']['properties']['status'].upper() == "RUNNING"
+            instance_count = item['properties']['activeDeployment']['properties']['deploymentSettings']['instanceCount']
             instances = item['properties']['activeDeployment']['properties']['instances']
             if instances is None:
                 instances = []
             up_number = len(
-                [x for x in instances if x['discoveryStatus'] == 'UP'])
-            item['Discovery Status'] = "UP( {} ), DOWN( {} )".format(
-                up_number, len(instances) - up_number)
+                [x for x in instances if x['discoveryStatus'].upper() == 'UP' or x['discoveryStatus'].upper() == 'OUT_OF_SERVICE'])
+            running_number = len(
+                [x for x in instances if x['status'].upper() == "RUNNING"])
+            item['Provisioning Status'] = item['properties']['activeDeployment']['properties']['provisioningState']
+            item['CPU'] = item['properties']['activeDeployment']['properties']['deploymentSettings']['cpu']
+            item['Memory'] = item['properties']['activeDeployment']['properties']['deploymentSettings']['memoryInGb']
+            item['Running Instance'] = "{}/{}".format(running_number, instance_count) if isStarted else "Stopped"
```

Comment:
it might cause problem if user replying on output json schema

Reply:
There is no swagger change happened in backend side. My understanding is the json output should be align with the swagger model, and this file only take effects on table output. Thus if a user rely on the json output, should not be affected. Is it right?